test/services/AddressLookupServiceSpec.scala

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import base.SpecBase
+import connectors.AddressLookupConnector
+import models.{AlfResponse, Warehouse}
+import models.backend.{Site, UkAddress}
+import org.mockito.MockitoSugar.{mock, when}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class AddressLookupServiceSpec extends SpecBase  {
+  val fixedUUID:String = "12"
+  val mockSdilConnector = mock[AddressLookupConnector]
+  implicit val hc = HeaderCarrier()
+  val service = new AddressLookupService(mockSdilConnector){
+    override def generateId: String = fixedUUID
+  }
+
```

Comment:
as a standard going forward as we are a public repo, can we use fake addresses please, not real addresses, as this is just unit tests with no validation, it can just be "foo bar"